app/styles/components/code.css

```
@@ -1,34 +1,34 @@
     .shiki {
         max-height: 30rem;
         padding: 1.15em 0.5em;
         padding-inline-end: 3em;
         margin-block: 0;
-        border: var(--border-width-thick) solid;
-        border-radius: var(--radius-md);
+        border: var(--cc-border-width-thick) solid;
+        border-radius: var(--cc-radius-md);
         overflow: auto;
 
         white-space: pre;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)